### ISSUE TITLE
内存溢出

### ISSUE THREAD

**Issue by shuxiao0312** [bubbliiiing/faster-rcnn-pytorch#27]
楼主你知不知道运行解冻后的epoch会出现killed的现象该怎么解决？
![image](https://user-images.githubusercontent.com/40878704/102967715-83157b00-452d-11eb-83b6-c3bfe60da516.png)

**Comment by shuxiao0312** [comment]
已经解决了~用这里面的解决办法就好[](https://pytorch.org/docs/stable/notes/faq.html)

**Comment by bubbliiiing** [comment]
啊这，有具体的解决方案吗，虽然我没遇到

**Comment by shuxiao0312** [comment]
> 啊这，有具体的解决方案吗，虽然我没遇到

就把loss += loss类似的语句改成loss += float(loss)就ok，这样子的目的是为了让loss变成不可自动求导，从而不会再训练循环中累积消耗内存~

**Comment by bubbliiiing** [comment]
我应该求个item就行，我可能是后面显示的时候才item
